src/test/java/tempest/ModuleViewTest.java

```diff
@@ -57,7 +69,7 @@ public void testAddingASession() {
                 break;
             }
         }
-
+        int previousSessions = testModule.getStudySessions().length;
```

Comment:
This last test fails to add sessions.

Reply:
In what way? The test doesn't do as described or the test fails when you run it?

Reply:
The test fails.
![image](https://user-images.githubusercontent.com/45072525/111148314-63697e00-8583-11eb-89f7-67d1f1598fa6.png)

Reply:
Strange its fine for me:

![Test Running](https://user-images.githubusercontent.com/65258846/111148736-e8549780-8583-11eb-8618-388e70d8a97b.png)

Reply:
Okay, that is weird